src/dto/index.ts

```diff
@@ -7,5 +7,11 @@ export interface ILyricsAndDetails {
   lyrics: string;
```

Comment:
In `src\api\lrclib.ts` I see you set this to `null`, so this type would need to be `string | null` - might be easier to use `undefined`